### PR TITLE
New Header and Navigation Menu for Home and Secondary Pages

### PR DESCRIPTION
**Pull request by mtribone**
Adds another partial for splitting up the menu for profile, primary, and secondary navigation. Updates the catalog controller to add 2 non-working links to the new mwnu for My Works and My Collections and adds partials for both. Includes the layout behavior for the controllers so that they pick up the override for the container classes from CHO and not Blacklight

Adds additional SCSS file for the header styles and a variable for the PSUL blue. Breaks the header/banner into two parts for banner, profile nav, and primary navigation. Updates the layout of the header for Bootstrap columns from mobile to desktop.

## Description

Customizes CHO header and navigation to make it visually more like Libraries and less like default Blacklight

Connected to #739 

## Changes

* Adds new graphics for header background and Libraries logo
* Splits up current menu into 2 sections. One menu for profile related item and another menu for the creation of works and collections.
* Semantic updates to cho layout partial. Replaces `<div>` with `<main>`, `role='banner'` for the div containing the branding
* New color variables for Libraries branding
* Adds nav action in Blacklight catalog controller for upcoming "My Works" and "My Collections" navigation items
* Overrides additional Blacklight partials to use our container classes for fluid layout
* Adds and removes Bootstrap classes where needed

### CODE REVIEW

**Comment by mtribone** [comment]
Screen Shot 2019-04-01 at 3 31 50 PM

**Comment by mtribone** [comment]
Screen Shot 2019-04-01 at 3 32 28 PM

**Comment by mtribone** [comment]
Screen Shot 2019-04-01 at 3 32 39 PM

File: app/cho/agent/resources_controller.rb [hunk 1]
```diff
@@ -4,6 +4,8 @@ module Agent
   class ResourcesController < ApplicationController
     include ValkyrieControllerBehaviors
 
+    helper LocalHelperBehavior, LayoutHelperBehavior
+
```
Comment (awead) [inline app/cho/agent/resources_controller.rb]: @mtribone we seem to need these everywhere. Would it make more sense to put them in `ApplicationController`? That way all other controllers would inherit from it. The only counterpoint would if we wanted to have a controller that _shouldn't_ inherit these helpers.

File: app/views/layouts/cho.html.erb [hunk 2]
```diff
@@ -19,6 +19,7 @@
     <title><%= render_page_title %></title>
     <%= opensearch_description_tag application_name, opensearch_catalog_url(format: 'xml') %>
     <%= favicon_link_tag %>
+    <%= stylesheet_link_tag 'https://use.typekit.net/uiy7uef.css' %>
```
Comment (awead) [inline app/views/layouts/cho.html.erb]: @mtribone is this a stable, versioned resource? What would happen if it went "away" or changed? It's also totally acceptable to punt on this right now.

File: app/assets/stylesheets/cho.scss [hunk 3]
```diff
@@ -1,3 +1,3 @@
 @charset "UTF-8";
 
-@import 'cho/variables', 'cho/blacklight', 'cho/data_dictionary_fields', 'cho/facets', 'cho/footer', 'cho/form', 'cho/scaffolds';
+@import 'cho/variables', 'cho/blacklight', 'cho/data_dictionary_fields', 'cho/facets', 'cho/footer', 'cho/form', 'cho/header', 'cho/scaffolds';
```
Comment (awead) [inline app/assets/stylesheets/cho.scss]: @mtribone how would you feel about new lines for each `@import`? It's not pretty, but it'll make future diffs easier to read. On the other hand, if this isn't going to change much or will go away, feel free to leave it as-is.

File: app/cho/agent/resources_controller.rb (hunk 1 above)
Comment (mtribone) [reply app/cho/agent/resources_controller.rb]: @awead Yeah it seemed like I needed to add one right after another when I was moving through the site.

File: app/views/layouts/cho.html.erb (hunk 2 above)
Comment (mtribone) [reply app/views/layouts/cho.html.erb]: This is the typeface for the header that the University uses as its 'official' typeface. We get it 'free' as part of the University's relationship with Adobe. It is possible that the contract might end in the future, but the CSS does have a fallback.

File: app/blacklight/catalog_controller.rb [hunk 4]
```diff
@@ -185,5 +185,8 @@ class CatalogController < ApplicationController
     # Configuration for autocomplete suggestor
     # config.autocomplete_enabled = true
     # config.autocomplete_path = 'suggest'
+
+    config.add_nav_action(:my_works, partial: 'blacklight/nav/my_works')
```
Comment (mtribone) [inline app/blacklight/catalog_controller.rb]: @awead These are here in prep for #810 but are non-functioning links.

File: app/views/layouts/cho.html.erb (hunk 2 above)
Comment (awead) [reply app/views/layouts/cho.html.erb]: @mtribone ok, that's fine for now then. My concern was we might want to import that css directly into the app so we don't have to rely on the external source, but it sounds like we want that external link so it's from a common location.

Comment (mtribone) [reply app/views/layouts/cho.html.erb]: @awead oh. I gotcha. let me see if that works.

**Pull request merged**